root/res/bandplans/netherlands.json

```diff
@@ -743,20 +761,32 @@
             "type": "amateur",
             "name": "Ch. 2 (60m) (Not NL)"
         },
+        {
+            "start": 5349400,
+            "end": 5351500,
+            "type": "amateur1",
+            "name": "60m Ham Band (Not NL)"
+        },		
         {
             "start": 5351500,
-            "end": 5366500,
+            "end": 5354000,
+            "type": "amateur",
+            "name": "| 60m Ham Band CW"
+        },
+        {
+            "start": 5354000,
+            "end": 5366000,
             "type": "amateur1",
-            "name": "60m Ham Band NL"
+            "name": "60m HAM Alle modes (USB)"
         },
         {
-            "start": 5357000,
-            "end": 5359900,
+            "start": 5366000,
+            "end": 5366500,
             "type": "amateur",
-            "name": "Ch. 3 (60m) (Not NL)"
+            "name": "60m HAM Weak NB modes |"
```

Comment:
remove pip character at the end

Reply:
I used pipes to mark the start and end of a band, when a band is divided in multiple sub-bands